### ISSUE TITLE
Code blocks look like textareas

### ISSUE THREAD

**Issue by joehoyle** [humanmade/altis-enhanced-search#113]
IMO the code blocks on the left in 
![Screenshot 2020-10-09 at 11 47 17 AM](https://user-images.githubusercontent.com/161683/95568981-37226200-0a25-11eb-86dd-b638051e5ac9.png) look too much like text inputs. I didn't read it all completely, but right away tried to click into them!

**Comment by roborourke** [comment]
Yeah that's been bugging me too! Not totally sure how best to style them differently
